dev/bin/shard-run.sh

```diff
@@ -15,6 +15,9 @@ mvn \
   -Dminikubeip=${MINIKUBE_IP} \
   -Dquarkus.http.port=1337 \
   -Devent-bridge.logging.json=false \
-  -Pminikube \
+  -Devent-bridge.k8s.orchestrator=minikube \
```

Comment:
Do we need to set [this](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/1067/files#diff-d34f9ea355d51ca9a2541fc7489e774973ebd6850c2dca32ff12f8c319bb6fb5L29) too?

I see it removed but not set elsewhere.

Reply:
no, its not required. Thanks for point it out.
I am removing it

Reply:
I was referring to `quarkus.kubernetes.deployment-target` that you have deleted from the `application.properties` file but not added back anywhere else. The "this" above is a link to where you delete the property.

Reply:
Now we are only have one single profile `kubernetes` so `quarkus.kubernetes.deployment-target` property is not required.